docs/source/intro/get_started.rst

```diff
@@ -1,69 +1,134 @@
 
-Installation
-============
+   *Install software directly into your local root directory.*
 
-A working AiiDA installation consists of three core components:
+   The prerequisite software can be installed using most package managers, including: apt, Homebrew, MacPorts, Gentoo and Windows Subsystem for Linux.
 
-* ``aiida-core``: The main python package and associated CLI ``verdi``.
-* `PostgreSQL <https://www.postgresql.org>`_: A service which manages the database where we store generated data.
-* `RabbitMQ <https://www.rabbitmq.com>`_: A service which manages communication with the processes that we run.
+   +++
 
-Each component may be installed separately, depending on your use case.
-Here we first provide the simplest approaches for installation on your local computer.
+   :link-badge:`intro:install:prerequisites,Prerequisites install,ref,badge-primary text-white`
+   :link-badge:`intro:install:aiida-core,aiida-core install,ref,badge-primary text-white`
 
-.. panels::
+   ---------------
+
+   :fa:`folder,mr-1` **Virtual Environment**
+
+   *Install software into an isolated directory on your machine.*
+
+   Environment managers such as `Conda <https://docs.conda.io>`__, `pipenv <https://pipenv.pypa.io>`__  and `venv <https://docs.python.org/3/library/venv.html>`__ create isolated Python environments.
+   It is advised that you install ``aiida-core`` into one of these managed environments.
+
+   +++
+
+   :link-badge:`intro:install:virtual_environments,Environments Tutorial,ref,badge-primary text-white`
+   :link-badge:`https://anaconda.org/conda-forge/aiida-core,aiida-core on Conda,url,badge-primary text-white`
+
+   ---------------
+
+   :fa:`cube,mr-1` **Containers**
+
+   *Use a pre-made image of all the required software.*
+
+   AiiDA maintains a `Docker <https://www.docker.com/>`__ image, which is particularly useful for learning and testing purposes.
+   It is a great way to quickly get started on the tutorials.
+
+   +++
+
+   :link-badge:`intro:install:docker,Docker Tutorial,ref,badge-primary text-white`
+   :link-badge:`https://hub.docker.com/r/aiidateam/aiida-core,aiida-core on DockerHub,url,badge-primary text-white`
+
+   ---------------
+
+   :fa:`cloud,mr-1` **Virtual Machines**
+
+   *Use a pre-made machine with all the required software.*
+
+   `Materials Cloud <https://www.materialscloud.org>`__ provides both downloadable and web based VMs,
+   also incorporating pre-installed Materials Science codes.
+
+   +++
+
+   :link-badge:`https://materialscloud.org/quantum-mobile,Quantum Mobile,url,badge-primary text-white`
+   :link-badge:`https://aiidalab.materialscloud.org,AiiDA lab,url,badge-primary text-white`
 
-    **Install from Conda**
+.. _intro:quick_start:
 
-    `Conda <https://docs.conda.io>`_ provides a cross-platform package management system, from which we can install all the basic components of the AiiDA infrastructure in an isolated environment:
+Quick Start Install
+===================
```

Comment:
I think it would be good to divide this guide into multiple steps, something like:

1. Install aiida-core
2. Initialize/setup(?) database
3. Setup profile
4. Start services
5. Stop services

Reply:
Fixed in e5c13be